plugins/graphiql/src/components/GraphiQLBrowser/GraphiQLBrowser.tsx

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
-import React, { useState, Suspense } from 'react';
-import { Tabs, Tab, makeStyles, Typography, Divider } from '@material-ui/core';
+import React, { Suspense, useState } from 'react';
+
+import { Progress } from '@backstage/core-components';
+import Divider from '@mui/material/Divider';
+import Tab from '@mui/material/Tab';
+import Tabs from '@mui/material/Tabs';
+import Typography from '@mui/material/Typography';
+import { Theme } from '@mui/material/styles';
+import makeStyles from '@mui/styles/makeStyles';
 import 'graphiql/graphiql.css';
-import { StorageBucket } from '../../lib/storage';
+
 import { GraphQLEndpoint } from '../../lib/api';
-import { BackstageTheme } from '@backstage/theme';
-import { Progress } from '@backstage/core-components';
+import { StorageBucket } from '../../lib/storage';
 
 const GraphiQL = React.lazy(() =>
   import('graphiql').then(m => ({ default: m.GraphiQL })),
 );
 
-const useStyles = makeStyles<BackstageTheme>(theme => ({
+const useStyles = makeStyles<Theme>(theme => ({
```

Comment:
not sure why explict `Theme` type import would be required here, but `yarn tsc` gave an error for using `theme.palette`. There might be something odd going on where the overrided `types` from `theme/v5/types.ts` won't effect this workspace.

Reply:
Hmm, interesting, it really shouldn't be